docker/pypi/wmagent/install.sh

```diff
@@ -144,6 +144,8 @@ export CRYPTOGRAPHY_ALLOW_OPENSSL_102=true
 export YUI_ROOT=$WMA_DEPLOY_DIR/yui/
 export PATH=\$WMA_INSTALL_DIR/bin\${PATH:+:\$PATH}
 export PATH=\$WMA_DEPLOY_DIR/bin\${PATH:+:\$PATH}
+export USER=\$(id -un)
+export WMA_USER=\$(id -un)
```

Comment:
I think all these imports are incorrectly escaping the $ sign. I will comment further in the final review comment.

Reply:
well the reallity is is that we should also escape the \( and \), because hat we need is to echo tis command as it is in the WMA_ENV_FILE, so that we delay the execution to be at the time the $WMA_ENV_FILE is actually sourced. We have no use of these if we  execute them t build time inside the `install.sh` script. At this stage noone knows  what the user to run the container would be.
I'll make the fix in a while

Reply:
Oh, now it makes sense. You are basically defining those environment variables with other environment variables that might not yet be defined.

Reply:
@amaltaro I think we can consider this conversation resolved, and move forward

Reply:
We cannot, this code needs to be fixed.